exec/java-exec/src/test/java/org/apache/drill/exec/physical/impl/lateraljoin/TestE2EUnnestAndLateral.java

```diff
@@ -169,13 +169,31 @@ public void testLeftLateral_WithFilterAndLimitInSubQuery() throws Exception {
   @Test
   public void testMultiUnnestAtSameLevel() throws Exception {
     String Sql = "EXPLAIN PLAN FOR SELECT customer.c_name, customer.c_address, U1.order_id, U1.order_amt," +
```

Comment:
If this fix allows execution for the query from the test, I think we should remove `EXPLAIN PLAN FOR` and check the result.

Reply:
@vvysotskyi This is an existing testcase. I just want to keep it intact. I cannot remove the EXPLAIN PLAN and use it for executing the query because this query will result in an execution error (unnest is not supported for non list types) and this is the right behaviour. For testing the usecase of this PR an additional testcase has been introduced which also tests for the correctness of the result.